src/bun.js/node/types.zig

```diff
@@ -2021,7 +2030,7 @@ pub const Path = struct {
     }
 
     pub fn getCwdWindowsU8(buf: []u8) MaybeBuf(u8) {
-        const u16Buf: [PATH_MAX_WIDE]u16 = undefined;
+        const u16Buf: [PathSize(u16).MAX]u16 = undefined;
```

Comment:
It could be yes. In the code though I to be consistent and if I'm referencing a PathSize() in places then thinking about oh that's a WPathBuffer with the correct size for that is extra overhead considering the pattern of `PathSize()`. MAX,MID,MIN. I did miss it being comptimed though.

Reply:
Turns out it was already comptime 💪

Reply:
Uses bun.WPathBuffer now.